layout/_partials/footer.swig

```diff
@@ -1,6 +1,9 @@
 <div class="copyright">{#
 #}{% set current = date(Date.now(), "YYYY") %}{#
-#}&copy; {% if theme.footer.since and theme.footer.since != current %}{{ theme.footer.since }} – {% endif %}{#
```

Comment:
Here is not `&mdash;`. Here is [n-dash](https://www.punctuationmatters.com/hyphen-dash-n-dash-and-m-dash/) (this: –, not that: —)

Reply:
ok, `&mdash`  will be displaced.